cassandane/Cassandane/Instance.pm

```diff
@@ -2853,6 +2853,14 @@ sub install_sieve_script
     die "Symlink does not exist: $sieved/defaultbc" if not -l "$sieved/defaultbc";
 
     xlog "Sieve script installed successfully";
```

Comment:
Does this commit really belong to that pull request? Is it related to the lock changes?